index.js

```diff
@@ -634,6 +658,8 @@ module.exports = {
     // Forbid unassigned imports.
     'import/no-unassigned-import': 'off',
     // Forbid named default exports.
-    'import/no-named-default': 'warn'
+    'import/no-named-default': 'warn',
+    // Forbid anonymous values as default exports
+    'import/no-anonymous-default-export': 'warn'
```

Comment:
I don't think we want this rule enabled; we export anonymous stuff all the time.